cmd/datamon/cmd/bundle_list.go

```diff
@@ -23,17 +25,18 @@ var BundleListCommand = &cobra.Command{
 		if err != nil {
 			logFatalln(err)
 		}
-		bundleDescriptors, err := core.ListBundles(params.repo.RepoName, remoteStores.meta, core.ConcurrentBundleList(params.core.ConcurrencyFactor))
-		if err != nil {
-			logFatalln(err)
-		}
-		for _, bd := range bundleDescriptors {
+		err = core.ListBundlesApply(params.repo.RepoName, remoteStores.meta, func(bundle model.BundleDescriptor) error {
```

Comment:
I think this works in go but it is ugly to read.. can we define a function before we pass it?

Reply:
ok if that is your preference.